src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -336,18 +335,14 @@ public record CertificateStoreOptions {
 			public string TrustedRootCertificateThumbprint { get; init; } = string.Empty;
 
 			internal static CertificateStoreOptions FromConfiguration(IConfigurationRoot configurationRoot) => new() {
-				CertificateStoreLocation = configurationRoot.GetValue<string>(nameof(CertificateStoreLocation)),
-				CertificateStoreName = configurationRoot.GetValue<string>(nameof(CertificateStoreName)),
-				CertificateSubjectName = configurationRoot.GetValue<string>(nameof(CertificateSubjectName)),
-				CertificateThumbprint = configurationRoot.GetValue<string>(nameof(CertificateThumbprint)),
-				TrustedRootCertificateStoreLocation =
-					configurationRoot.GetValue<string>(nameof(TrustedRootCertificateStoreLocation)),
-				TrustedRootCertificateStoreName =
-					configurationRoot.GetValue<string>(nameof(TrustedRootCertificateStoreName)),
-				TrustedRootCertificateThumbprint =
-					configurationRoot.GetValue<string>(nameof(TrustedRootCertificateThumbprint)),
-				TrustedRootCertificateSubjectName =
-					configurationRoot.GetValue<string>(nameof(TrustedRootCertificateSubjectName))
+				CertificateStoreLocation = configurationRoot.GetString(nameof(CertificateStoreLocation)),
+				CertificateStoreName = configurationRoot.GetString(nameof(CertificateStoreName)),
+				CertificateSubjectName = configurationRoot.GetValue<string>(nameof(CertificateSubjectName)) ?? string.Empty,
```

Comment:
any reason this one is different to the others?

Reply:
Hmm slipped through wile testing I'm afraid, fixed.